### DISCUSSION TITLE
2009/05/26/%E6%8B%BC%E5%9B%BE%E6%B8%B8%E6%88%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#267] [Announcements]
# 2009/05/26/%E6%8B%BC%E5%9B%BE%E6%B8%B8%E6%88%8F

大鸟送了我一套纸质拼图游戏，于是这两天有个好玩具玩。尽管已经二十年没玩这东西了，不过我当建筑工人的天赋还在，不用看图纸就把所有的零件都搭起来了。实际上，现在的拼图玩具跟我小时候玩的那种已经完全不同了。现在的拼图材质大为提高，印刷也很精美，更重要的是不再需要胶水了，所有的零件只通过插接就可以牢固的拼装在一起。

https://blog.qizhen.xyz/2009/05/26/%E6%8B%BC%E5%9B%BE%E6%B8%B8%E6%88%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋阳
2009年07月14日 10:52 编辑
看起来很好玩的！好想玩，下次也去买几幅拼拼！

回复
Shi
2009年06月17日 15:52 编辑
Ruan大侠果然心灵手巧的很~~~话说，几个月不见，好黑啊。而且还留了胡子了，这形象改变的快的嘛 “ ）

回复
Shaodong
2009年06月2日 10:56 编辑
呵呵，这手工不错哈，再问下，您老的书。。。什么时候出啊。。。

回复
Qizhen
2009年05月27日 08:55 编辑
黑嘛，可能是因为每天骑车上班晒得。不过，到了我这个年龄，健康比外貌就重要得多了，还是黑点好啊。

回复
xiaoji
2009年05月26日 21:38 编辑
我的1500片拼了快一年了，看来还是一气呵成的好呀

回复
Haiying
2009年05月26日 21:32 编辑
你咋这么黑了呢 hehe

回复
